### DISCUSSION TITLE
Get most variable peaks from a list to run motif analysis

### DISCUSSION THREAD

**Discussion by feanaros** [stuart-lab/signac#1464] [Q&A] [answered]
Hello, im wondering if there Is a way to avoid this step of analysis: 
DefaultAssay(gdt17) <- "ATAC"

da_motif <- FindMarkers(
  object = gdt17,
  ident.1 = 0,
  ident.2 = 7,
  only.pos = TRUE,
  test.use = 'LR',
  min.pct = 0.05,
  latent.vars = 'nCount_ATAC'
)

sig_peak <- rownames(da_motif[da_motif$p_val_adj <.05,])

And use as input  list of peaks I'm interested in. The problem Is that I didn't get interesting results. How can I select most variable peaks from that list? There Is a way to do It? And about the BG, i would to use all the cluster of my dataset. It makes sense if I use as background all the peaks of my dataset?

**Comment by timoast** [comment, marked as answer]
You can certainly use any set of peaks as input to `FindMotifs()`, this doesn't need to come from a differential test. If you want to select the most variable peaks from a list, you could compute the accessibility fold change between two populations of cells using the `FoldChange()` function.

For the background, you should try to match this according to how you selected the set of peaks in the test set. The `FindMotifs()` function will compare the frequency of motifs within the DNA sequence of the test peaks vs the background set.
